lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1365,8 +1371,9 @@ def unit_ip(self) -> str:
         if bind_address:
             logger.warning("Returning IP %s", str(bind_address))
             return str(bind_address)
-        logger.warning("No address found")
-        return ""
+        else:
+            logger.warning("No address found")
```

Comment:
IMO this log message is not informative enough to the end use, and should definitely not be a warning if we recover from it later. And if we do not recover, it's ERROR material ;-)